ja/xml/ha_clvm.xml

```diff
@@ -306,7 +306,7 @@
     </step>
    </procedure>
    <para>
-    LVM2に関する詳細情報が必要な場合は、『SUSE Linux Enterprise Server <phrase role="productnumber"><phrase os="sles">12 SP5</phrase></phrase> <citetitle>ストレージ管理ガイド</citetitle>』(<link xlink:href="dsc;/sles-12/html/SLES-all/cha-lvm.html"/>)を参照してください。
+    LVM2に関する詳細情報が必要な場合は、『SUSE Linux Enterprise Server <phrase role="productnumber"><phrase os="sles">12 SP5</phrase></phrase> <citetitle>ストレージ管理ガイド</citetitle>』(<link xlink:href="dsc;/sle-ha/12-SP5/single-html/SLE-HA-guide/#book-sleha"/>)を参照してください。
```

Comment:
Please revert this change - see https://github.com/SUSE/doc-sleha/pull/199#discussion_r844098808 for reason why.